package.json

```diff
@@ -60,6 +60,8 @@
     "jsdom": "^20.0.0",
     "jszip": "^3.10.1",
     "jwt-decode": "3.1.2",
+    "linkify-react": "^3.0.4",
+    "linkifyjs": "^3.0.5",
```

Comment:
Is `linkifyjs` also needed?

Reply:
It's a peer-dependency, so unfortunately yes :'(